GLTF/src/GLTFMaterial.cpp

```diff
@@ -914,12 +956,10 @@ GLTF::MaterialPBR* GLTF::MaterialCommon::getMaterialPBR(GLTF::Options* options)
 			material->specularGlossiness->specularGlossinessTexture = texture;
 		}
 		if (values->shininess) {
-			if (values->shininess[0] < 1.0) {
-				material->specularGlossiness->glossinessFactor = values->shininess;
-			}
-			else {
-				material->specularGlossiness->glossinessFactor = new float[1] {1.0};
+			if (values->shininess[0] > 1.0) {
+                values->shininess = new float[1] {1.0};
```

Comment:
I think I originally did this to avoid modifying `values`.

If there are no ownership issues here why not just set `values->shininess[0] = 1.0`? If there is an ownership issue, maybe always create a new `float[1]` here.